### PR TITLE
[master]Fix completions in named args with included record params context

### LINKED ISSUE #39296: [Bug]: No completion items for record fields of record typed named argument
### Description

image

The `secureSocket` argument is of the following type. 
```Ballerina
public type ListenerSecureSocket record {|
    crypto:KeyStore|CertKey key;
    record {|
        VerifyClient verifyClient = REQUIRE;
        crypto:TrustStore|string cert;
    |} mutualSsl?;
    record {|
        Protocol name;
        string[] versions = [];
    |} protocol?;
    record {|
        CertValidationType 'type = OCSP_STAPLING;
        int cacheSize;
        int cacheValidityPeriod;
    |} certValidation?;
    string[] ciphers = ["TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
                        "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA",
                        "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA", "TLS_DHE_RSA_WITH_AES_128_CBC_SHA",
                        "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
                        "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"];
    boolean shareSession = true;
    decimal handshakeTimeout?;
    decimal sessionTimeout?;
|};

```

The fields are not suggested as completion items. 

### Steps to Reproduce

Mentioned above

### Affected Version(s)

2201.3.2

### OS, DB, other environment details and versions

_No response_

### Related area

-> Editor

### Related issue(s) (optional)

_No response_

### Suggested label(s) (optional)

_No response_

### Suggested assignee(s) (optional)

_No response_

### PR DESCRIPTION
**Pull request by malinthar**
## Purpose
$subject

Fixes #39296 

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by malinthar** [comment]
Blocked by https://github.com/ballerina-platform/ballerina-lang/issues/39296#issuecomment-1399898536

**Comment by IMS94** [comment]
@malinthar https://github.com/ballerina-platform/ballerina-lang/issues/39378 is now resolved. We can merge this PR.

**Review by IMS94 — APPROVED** [review]

File: language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config77.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "position": {
+    "line": 61,
+    "character": 66
+  },
+  "source": "expression_context/source/mapping_expr_ctx_source77.bal",
+  "description": "",
```
Comment (mindula) [inline language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config77.json:7]: Shall we add a description?

File: language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config78.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+  "position": {
+    "line": 66,
+    "character": 71
+  },
+  "source": "expression_context/source/mapping_expr_ctx_source78.bal",
+  "description": "",
```
Comment (mindula) [inline language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config78.json:7]: Add a description

**Review by IMS94 — APPROVED** [review]

**Pull request merged**